internal/pkg/config/workload.go

```diff
@@ -186,7 +186,7 @@ func (s *Store) ListServices(appName string) ([]*Workload, error) {
 func (s *Store) ListJobs(appName string) ([]*Workload, error) {
 	wklds, err := s.listWorkloads(appName)
 	if err != nil {
-		return nil, fmt.Errorf("read service configuration for application %s: %w", appName, err)
+		return nil, fmt.Errorf("read job configuration for application %s: %w", appName, err)
```

Comment:
do we need this error wrapping?

Reply:
I think it's still worth because it is to avoid using "workload" term in error msg. For example: `read service configuration for application chicken: invalid character 'o' looking for beginning of value`